book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] To correctly complete the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` classes using the intermediate class `Rental`, kindly ensure that `Getter` and `Setters` for `@rentals` sre present i.e. `attr_accessor :title, :author, :rentals`